active/teachingtip/TeachingTipSpec.md

```diff
@@ -101,7 +101,7 @@ public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
 
 ### Non-pointing tips
 
-Not all tips relate to an element onscreen. For these scenarios, do not set the Target property and the teaching tip will instead display relative to the edges of the app window. However, a teaching tip can have the pointer removed while retaining placement relative to a UI element by setting the PointerMode property to "Off". 
+Not all tips relate to an element onscreen. For these scenarios, do not set the Target property and the teaching tip will instead display relative to the edges of the UWP window. However, a teaching tip can have the pointer removed while retaining placement relative to a UI element by setting the PointerMode property to "Off". 
```

Comment:
I don't think "UWP" window is an improvement.  Is this in reference to the islands question?

Reply:
@MikeHillberg, it is. Is this control unqiue enough that perhaps I should add a section for working with islands?

Reply:
No, I think we just need a consistent overall strategy for how we talk about these out-of-bounds flyouts.  Can you just call this out as an issue for now?

Reply:
In other places we've used the word "root" (it could be the root of a window or the root of an island).  So: "...display relative to the edges of the root...".  Maybe that's the best word to start with for now at least.

Reply:
Committed the the fix [here](https://github.com/Microsoft/microsoft-ui-xaml-specs/commit/64e20c30af4f20023420b71d2d34566aedf347e3).

Reply:
And the committed fix you linked to, @SavoySchuler , it calls it a "root window" - but with XAML islands that seems confusing.  The word "window" isn't necessary, it may add confusion with the top-level window (which isn't what you meant).

Reply:
@MikeHillberg, any preference between "root" or "XamlRoot"?

Reply:
XamlRoot sounds good

Reply:
Fixed!